api/src/main/java/ca/bc/gov/educ/api/institute/struct/v1/School.java

```diff
@@ -44,6 +44,10 @@ public class School extends BaseRequest implements Serializable {
   @Size(max = 255)
   private String website;
 
+  @Size(max = 255)
```

Comment:
This should be changed to max 10.

Reply:
Already bagged this one in https://github.com/bcgov/EDUC-INSTITUTE-API/pull/128/commits/0366e84ce3207b7fee7f5d98cb285843804ed7ce :D